spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -288,7 +289,7 @@ describe('document-capture/components/acuant-capture', () => {
       expect(document.activeElement).to.equal(button);
     });
 
-    it('shows error if capture fails: latest version of Acuant SDK', async () => {
+    it('shows a generic error if camera starts but cropping error occurs', async () => {
```

Comment:
This comment might not be applicable if you add another test as discussed in the previous comment, but:

if I'm reading it right this `it` statement doesn't match [the test expectation](https://github.com/18F/identity-idp/blob/654c78b5c8f466f75ff64013209dfbb189840517/spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx#L315-L320).

Reply:
Good catch. Fixed that in this [commit](https://github.com/18F/identity-idp/pull/8860/commits/1fab49d1a4eb95f607932ea24910d55cd327bf0a).